css/style.css

```
@@ -76,22 +76,89 @@ button {
 
 header {
   width: 1160px;
+  min-height: 881px;
   margin: 0 auto;
   display: flex;
   position: relative;
+  background-image: url("../img/light-back.png");
+  background-repeat: no-repeat;
+  background-position: center;
 }
 
 
 .header-logo {
   margin-right: 27px;
-  padding-top: 33px;
+  padding-top: 9px;
 }
 
 .header-nav {
   min-width: 350px;
-  padding-top: 33px;
   display: flex;
   justify-content: space-between;
+  align-items: start;
+  margin-top: 30px;
+}
+
+.header-nav__element {
+  display: inline-block;
+  position: relative;
+  height: 18px;
+  padding-bottom: 2px;
+  padding-left: 13px;
+  padding-right: 13px;
+}
+
+.header-nav__link {
+  color: #ffffff;
+  border-bottom: 2px solid #9eb3a8;
+}
+
+.header-nav__element:hover {
+  background-color: #ffffff;
+  border-bottom: none;
+  border-radius: 50px;
+}
+
+.header-nav__element:hover .header-nav__link {
+  border: none;
+  color: #000000;
+}
+
+.header-nav__element:hover .catalog-submenu {
+  display: none;
```

Почему не 
```suggestion
  display: block;
```
?
Не доделано ещё?

---

да в процессе, чтоб не мешались пока